MapboxNavigation/CarPlayMapViewController.swift

```diff
@@ -24,7 +26,9 @@ class CarPlayMapViewController: UIViewController, MGLMapViewDelegate {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-
+        
+        styleManager = StyleManager(self)
+        styleManager.styles = [CarPlayDayStyle(), CarPlayNightStyle()]
```

Comment:
@1ec5 should we consider creating a `BaseViewController` that assigns these styles and any other shared configurations for the view controllers? I see some repetition with styles for both `CarPlayMapViewController` and `CarPlayNavigationViewController`.